shell/public/tokentemplate.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Sandstorm template</title>
+    <style>
+body {
+  margin: 0;
+  padding: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
+  border: 0;
+}
+pre {
+  margin: 0;
+  padding: 0;
+}
+    </style>
+  </head>
+  <body>
+    <pre id="text"></pre>
+<script>
+// Templates are namespaced in localStorage by the prefix "template"
+var templateToken = "template" + window.location.hash.substring(1);
```

Comment:
How about "offerTemplate", to be somewhat more specific? Filename could be `offer-template.html`.